### ISSUE TITLE
rotate coordinates in metadata file

### ISSUE THREAD

**Issue by TomMelt** [nextsimhub/domain_decomp#46]
Currently `decomp` uses the following coordinate layout:

```
O┌───►Y
 │     
 │     
X▼     
```

But this causes a problem for nextsim which needs to know the concept of Bottom, Right, Top & Left neighbours (BLTR).

In order for the two codes (`decomp` and `nextsim`) to agree on which neighbour is BLTR, we need to use the same coordinate system. @timspainNERSC mentioned that we should use axis as in mathematics, as follows:

```
Y▲     
 │     
 │     
O└───►X
```

**Comment by jwallwork23** [comment]
@TomMelt I guess this was resolved by your recent changes?

**Comment by TomMelt** [comment]
@jwallwork23 , that's correct. It was fixed by #58.
